processor/opentelemetry-processor-baggage/README.rst

```diff
@@ -20,3 +20,32 @@ Do not put sensitive information in Baggage.
 
 To repeat: a consequence of adding data to Baggage is that the keys and
 values will appear in all outgoing HTTP headers from the application.
+
+## Usage
+
+Add the span processor when configuring the tracer provider.
+
+To configure the span processor to copy all baggage entries during configuration:
+
+```python
+from opentelemetry.processor.baggage import BaggageSpanProcessor, ALLOW_ALL_BAGGAGE_KEYS
+
+tracer_provider = TracerProvider()
+tracer_provider.add_span_processor(BaggageSpanProcessor(ALLOW_ALL_BAGGAGE_KEYS))
```

Comment:
Does this need to be a required arg or could this just be the default value?

Reply:
For consistency with other SDK implementations, I went went with a required arg to help make the processor secure by default by requiring the developer to set the allows baggage keys. If it were a default parameter, it's easy to add the processor and not know all keys are being copied.